docs/index.rst

```diff
@@ -0,0 +1,37 @@
+.. index::
+   pair: watertap-seto;Home
+
+Welcome to WaterTAP-SETO's documentation!
+======================================
+
+WaterTAP is a National Alliance for Water Innovation (NAWI) funded initiative to create an open-source water treatment model library
+that is compatible with the IDAES Platform, an advanced process systems engineering tool developed by the U.S. Department of Energy.
+
+WaterTAP-SETO is an extension of WaterTAP that incorporates solar energy models for solar-driven desalination systems.
+
+Collaborating Institutions
+--------------------------
+
+The WaterTAP-SETO development team is composed of researchers from:
+
+* National Renewable Energy Laboratory
+* National Energy Technology Laboratory
+
+Content
+-------
+
+.. toctree::
+   :maxdepth: 2
+
+   getting_started
+   tutorials/index
+   how_to_guides/index
+   technical_reference/index
+   background/index
+   license
+   copyright
+
+Indices and tables
+------------------
+
+* :ref:`genindex`
```

Comment:
unsure where this will come from

Reply:
I think this is generated automatically from your doc structure (`genindex` -> `gen`erated `index`?)

Reply:
Ah ok. I just copied this from WaterTAP.